crates/holochain_conductor_api/src/app_interface.rs

```diff
@@ -164,12 +165,28 @@ pub enum AppRequest {
     /// [`AppResponse::CloneCellEnabled`]
     EnableCloneCell(Box<EnableCloneCellPayload>),
 
-    /// Info about networking processes
+    /// Retrieve network metrics for the current app.
+    ///
+    /// Identical to what [`AdminRequest::DumpNetworkMetrics`](crate::admin_interface::AdminRequest)
+    /// does, but scoped to the current app.
+    ///
+    /// If `dna_hash` is not set, metrics for all DNAs in the current app are returned.
     ///
     /// # Returns
     ///
-    /// [`AppResponse::NetworkInfo`]
-    NetworkInfo(Box<NetworkInfoRequestPayload>),
+    /// [`AppResponse::NetworkMetrics`]
+    NetworkMetrics {
+        /// If set, limits the metrics dumped to a single DNA hash space.
+        #[serde(default)]
+        dna_hash: Option<DnaHash>,
+
+        /// Whether to include a DHT summary.
+        ///
+        /// You need a dump from multiple nodes in order to make a comparison, so this is not
```

Comment:
This comment is not clear to me. What is meant by "comparison" in this context? Is the comment trying to say that this information is only useful if you get that information from multiple nodes and can compare them?

Reply:
I don't really know what somebody would do with a single picture of the DHT. You could check what time segments have hashes I guess but at this level you don't really know what timestamps that corresponds to so it's not very interesting. See https://github.com/holochain/kitsune2/blob/main/crates/dht/src/dht/snapshot.rs#L73 for what I mean by comparing.

I'm thinking that it'd be possible to see from a dump whether two apps are in sync, if you take the dumps at about the same time.